source/gems/dry-component/booting-a-dependency.html.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Booting a Dependency
 layout: gem-single
+name: dry-component
```

Comment:
right, this should be recursive, and we probably want to add some wrapper around plain middleman resources so we can just do `current_page.name`